src/duplocloud/args.py

```diff
@@ -245,3 +245,17 @@
             help='The content directory for a website.',
             default='dist',
             env='DUPLO_CONTENT')
+
+SSM_PARAM_TYPE = Arg("parametertype", "-ptype",
```

Comment:
I assume you're referencing param_content, not param_type.  Param_type must be String, SecureString, or StringList.  I did check and there are no existing arbitrary string arguments unless you do something confusing like reuse something.  I created param_content, so that can be reused for subsequent raw text arguments like secretsmanager secrets.  I originally had it named RAW_CONTENT to highlight it should be used for future generic strings, but I thought PARAM_CONTENT was clearer.  I'm open to renaming this.